### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by YV-00** [codewars/codewars.com#2970] [Q&A]
Hi all. My account codewars is blocked and completely reset. Why?

**Comment by hobovsky** [comment]
Knowing what account it is would help a lot.

Can you give a link to your user profile?

**Reply by hobovsky** [reply]
I did not look at all solutions, but just looking at a few blue and purple ones,  i think the restriction did not happen without a reason.

**Reply by YV-00** [reply]
So what's the problem?

**Reply by hobovsky** [reply]
The problem seems to be the fact that reward (honor points and rank) for quite a few difficult kata was received by submitting code written by someone else.

**Reply by YV-00** [reply]
On this site, 80 percent of the code is written by someone else :)

**Reply by YV-00** [reply]
Extremely unfair and not objective decision.

**Reply by YV-00** [reply]
Thanks for explaining the possible reason.
